src/components/Reviews.jsx

```diff
@@ -0,0 +1,39 @@
+import { useState, useEffect } from 'react'
+import { fetchReviews } from './Apis'
+
+const Reviews = () => {
+
+    const [reviews, setReviews] = useState([]);
+
+    useEffect(() => {
+        fetchReviews().then((response) => {
+            setReviews(response.reviews)
+        })
+    }, [])
+
+    console.log(reviews)
+
+    return (
+        <>
+            <p>Game Reviews</p>
+                <section id="reviews">
```

Comment:
Rather than a section and a  list of articles you could use a main, ul and some li's (could be a little more semantic)

Reply:
Yes - wasn't sure how much info I wanted to include for each of the reviews... decided to go for a briefer content on this page but forgot to change out of using article tag. Will go change that next.